mundane-dsv/src/main/scala/com/ambiata/mundane/dsv/DelimitedValuesParser.scala

```diff
@@ -0,0 +1,68 @@
+package com.ambiata.mundane
+package dsv
+
+import scala.collection.JavaConversions._
+
+import org.apache.commons.csv.{CSVParser, CSVFormat}
+
+import scalaz._, Scalaz._
+
+/**
+ * Create a parser for delimited values based on
+ * different implementations and a choice of
+ *
+ *  - delimiter
+ *  - character used for quoting fields
+ *  - character used to escaped control characters (delimiter, quoteCharacter)
+ *
+ */
+object DelimitedValuesParser {
```

Comment:
Can we not call this delimited. My preference is for CSV - the standard has moved past the 'C' standing for comma. I really want to differentiate with what we call delimited which is "cleaned" data that should only be split.